superdesk/tests/__init__.py

```diff
@@ -479,8 +479,12 @@ async def setup_db_user(context, user):
 
         user["password"] = original_password
         auth_data = json.dumps({"username": user["username"], "password": user["password"]})
+
+        context.headers.append(["Content-Type", "application/json"])
```

Comment:
Although most requests are, not every request is "application/json". Where is this causing an issue, might need to be fixed there instead

Reply:
Fixed